app/views/support_interface/application_forms/nationalities/edit.html.erb

```diff
@@ -0,0 +1,41 @@
+<% content_for :browser_title, title_with_error_prefix('Edit applicant nationality', @nationalities_form.errors.any?) %>
```

Comment:
Could we make this `candidate` instead? 

I think they are synonymous within the service and so I think we should stick to just using one to reduce mental overheads.

Reply:
Agree in principle, but this is a continuation of language already in use in the Support interface.

@paulrobertlloyd we're calling candidates "applicants" in the Support and Provider interfaces. Is that how we should continue to refer to them, or should we standardise on "candidate"?